### DISCUSSION TITLE
How can compile_commands.json be generated when compiling a library with vcpkg?

### DISCUSSION THREAD

**Discussion by javenleeCH** [microsoft/vcpkg#43027] [Q&A]
How can compile_commands.json be generated when compiling a library with vcpkg?

**Comment by Neumann-A** [comment]
You are aware that vcpkg won't compile anything if the library has been build and is in the cache? You can however always put CMAKE_EXPORT_COMPILE_COMMANDS into VCPKG_CMAKE_CONFIGURE_OPTIONS within a triplet (or set the env variable on linux/osx. On Windows it requires setting it in the triplet or toolchain)
